### PR TITLE
updating docker compose to ease local development

### PR DESCRIPTION
**Pull request by xoscar**
This PR eases the local development and usage for the tracetest app.

## Changes

- Includes a way for containers to talk to the host machine.
- Removes the requirement to run the pokeshop services all of the time
- Exposes the rest of the jaeger ports.

## Fixes

- Running tracetest for local development.

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: local-config/config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+http:
+  prefix: someprefix
```
Comment (schoren) [inline local-config/config.yaml:2]: this looks like it shouldn't be here

File: local-config/config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+maxWaitTimeForTrace: 10s
+postgresConnString: "host=127.0.0.1 user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+jaegerConnectionConfig:
+  endpoint: localhost:16685
+  tls:
+    insecure: true
+googleAnalytics:
+  enabled: false
```
Comment (mathnogueira) [inline local-config/config.yaml:10]: This config file is in an old format. Take a look at the new format: https://github.com/kubeshop/tracetest/blob/main/local-config/config.tests.yaml

File: local-config/config.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+http:
+  prefix: someprefix
+maxWaitTimeForTrace: 10s
+postgresConnString: "host=127.0.0.1 user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+jaegerConnectionConfig:
+  endpoint: localhost:16685
```
Comment (schoren) [inline local-config/config.yaml:6]: unless really neccesary, I'd try to use the docker internal network as much as possible. I imagine tracetest should be able to talk with jaeger using the `jaeger:16685` url

File: local-config/config.yaml (hunk 2 above)
Comment (schoren) [inline local-config/config.yaml:10]: I think we should have GA enabled by default for new people

File: docker-compose.yaml [hunk 4]
```diff
@@ -57,14 +55,18 @@ services:
...
-      - "16685:16685"
+      - 5775:5775/udp
+      - 6831:6831/udp
+      - 6832:6832/udp
+      - 5778:5778
+      - 4318:4318
+      - 16685:16685
+      - 16686:16686
+      - 14268:14268
+      - 9411:9411
```
Comment (schoren) [inline docker-compose.yaml:69]: is it necessary to expose ALL jaeger ports?

File: local-config/config.yaml (hunk 2 above)
Comment (kdhamric) [reply local-config/config.yaml:10]: We probably need to rename it now that we are using Segment. Perhaps 'collectTelemetry:' instead?

File: local-config/config.yaml (hunk 1 above)
Comment (xoscar) [reply local-config/config.yaml]: We are not even using this file so I just deleted it 😅

File: local-config/config.yaml (hunk 2 above)
Comment (xoscar) [reply local-config/config.yaml]: We are not even using this file so I just deleted it 😅

File: docker-compose.yaml (hunk 4 above)
Comment (xoscar) [reply docker-compose.yaml]: Updated 😄

**Review by schoren — APPROVED** [review]
LGTM!

**Pull request merged**